### PR TITLE
Add sandbox documentation

### PR DESCRIPTION
**Pull request by bghgary**
The original page was created by ChatGPT with the prompt `Create a doc page for the Babylon.js sandbox`. I hand modified it a lot, especially the advanced section.

### CODE REVIEW

**Comment by bjsplat** [comment]
snapshot for this PR available at:

https://snapshots-cvgtc2eugrd3cgfd.z01.azurefd.net/DOCS/refs/pull/1240/merge

File: content/toolsAndResources/sandbox.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+- `autoRotate` - Set to `true` to turn on auto rotation of the active camera. Auto rotation is off by default.
+- `camera` - The active camera index to use from the loaded scene.
+- `cameraPosition` - The comma-separated `x,y,z` position of the active camera.
+- `clearColor` - The clear color for the loaded scene in hex form (e.g., `FFFFFF` for pure white).
+- `environment` - The environment URL to load for glTF assets. This parameter is ignored for other asset formats.
+- `kiosk` - Set to `true` to hide the toolbar. The toolbar is visible by default.
+- `skybox` - Set to `false` to not create a skybox for glTF assets. Skyboxes are created by default for glTF assets. This parameter is ignored for other asset formats.
+- `toneMapping` - The tone mapping (`Standard`, `ACES`, `KHR_PBR_Neutral`) to use for the loaded scene.
+
+#### Examples
```
Comment (RaananW) [inline content/toolsAndResources/sandbox.md:71]:
```suggestion
## Examples
```

**Review by RaananW — APPROVED** [review]
Approve (with suggestions :-) )

File: content/toolsAndResources/sandbox.md (hunk 1 above)
Comment (bghgary) [reply content/toolsAndResources/sandbox.md:71]: Why? These are examples of the URL parameters.

**Review by RaananW — APPROVED** [review]

**Pull request merged**